internal/executor/build/command.go

```diff
@@ -0,0 +1,31 @@
+package build
+
+import (
+	"github.com/cirruslabs/cirrus-ci-agent/api"
+	"github.com/cirruslabs/cirrus-cli/internal/executor/build/taskstatus"
+	"sync"
+)
+
+type Command struct {
+	status taskstatus.Status
```

Comment:
`taskstatus` looks a bit weird here. It can also be of `New` value. 🤔 What do you think about introducing `commandstatus`?

Reply:
>It can also be of `New` value.

What's wrong with that?

Reply:
`New` just sounds a bit weird. See how it's [defined in GraphQL](https://github.com/cirruslabs/cirrus-ci-web/blob/master/schema.graphql#L644-L651). I think for consistency we should also rename New to [Created](https://github.com/cirruslabs/cirrus-ci-web/blob/master/schema.graphql#L691) in a separate PR.

Reply:
See https://github.com/cirruslabs/cirrus-cli/pull/29/commits/1ca413f6d7ae530b7f92d9caabda2d2bbd26a661.